src/main/java/NewExceptions/EmptyArgumentException.java

```diff
@@ -0,0 +1,4 @@
+package NewExceptions;
+
+public class EmptyArgumentException extends Exception {
+}
```

Comment:
Try to avoid empty methods. Perhaps, add methods to handle the exception. Same goes for the other exception classes.